jujugui/static/gui/src/app/components/generic-input/generic-input.js

```diff
@@ -288,6 +288,10 @@ GenericInput.defaultProps = {
   type: 'text'
 };
 
+if (module) {
```

Comment:
Some components are used elsewhere in the app and those haven't yet been converted. So as an interim step some components support both requires and YUI. This will not be permanent.